sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -26,11 +26,12 @@ import org.apache.spark.sql.catalyst.expressions.aggregate.AggregateExpression
 import org.apache.spark.sql.execution.{SparkPlan, UnaryExecNode}
 import org.apache.spark.sql.types.{CalendarIntervalType, DateType, IntegerType, TimestampType}
 
-abstract class WindowExecBase(
-    windowExpression: Seq[NamedExpression],
-    partitionSpec: Seq[Expression],
-    orderSpec: Seq[SortOrder],
-    child: SparkPlan) extends UnaryExecNode {
+abstract class WindowExecBase extends UnaryExecNode {
+  def windowExpression: Seq[NamedExpression]
+
```

Comment:
super nit: I feel its better to avoid this blank line along with the others: https://github.com/apache/spark/pull/27368/files#diff-193703359b7f77e83814f1dece93be5fR28-R31

Reply:
Yea, thanks for the suggestion!